wiki/en/en-Tips-Tricks-More.md

```diff
@@ -53,11 +53,20 @@ function loadserverstat ( $statfilename )
 
 You can save and restore the mix you have for your band rehearsals (fader, mute, pan, solo etc.) and load these any time (even while you are playing). Do this with "File > Save Mixer Channels Setup" in your client and load them using "Load Mixer Channels Setup" (or drag/drop them to the mixer window).
 
+## Delegate mixing to a mix master using the --mastermix server mode
+
+By starting a server instance with the `--mastermix` flag, the first client to connect will become the mix master.
+
+Subsequent joiners will be able to control their own as well as the mix master's pan and gain, but nobody else's (clients after version TODO will see their faders and buttons grayed out). Instead, they will hear the pan and gain value as set by the mix master.
```

Comment:
We have usually kept the docs relevant to the current version and then update them once the new version is out.

Reply:
> Or does it mean they can control the mix master's channel and theirs, but nobody else's?

Exactly this. I know it might sound weird, but it's exactly what we need in our choir rehearsals, since 
1. everybody still has individual needs for how low the conductor/click is
2. the conductor mutes himself so he wouldn't be able to mix his own channel in the master mix